README.md

```diff
@@ -1 +1,186 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Hello Microverse <a name="about-project"></a>
+
+I created my portfolio using HTML, CSS with Linters and Figma templates.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+3. Linters
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- Since your live demo is not currently available i will kindly suggest you remove this section from your readme to make it look professional until your project is successfully deployed :+1: